utils.py

```diff
@@ -136,20 +136,21 @@ def read_params(ctx):
 
 
 def generate_metadata(data_dir,
+                      meta_dir,
                       masks_overlayed_dir,
-                      masks_overlayed_eroded_dir,
                       process_train_data=True,
                       process_validation_data=True,
                       process_test_data=True,
                       public_paths=False,
-                      competition_stage=1):
+                      competition_stage=1,
+                      ):
     if competition_stage != 1:
         raise NotImplementedError('only stage_1 is supported for now')
 
     def _generate_metadata(dataset):
         assert dataset in ["train", "test", "val"], "Unknown dataset!"
-        df_metadata = pd.DataFrame(columns=['ImageId', 'file_path_image', 'file_path_mask', 'file_path_mask_eroded',
-                                            'is_train', 'is_valid', 'is_test', 'n_buildings'])
+        # df_metadata = pd.DataFrame(columns=['ImageId', 'file_path_image', 'file_path_mask', 'file_path_mask_eroded',
```

Comment:
@taraspiotr Why is this commented out?

Reply:
@jakubczakon ups, forgot to delete it, df is created later